vercel/resource_deployment_test.go

```diff
@@ -109,7 +109,8 @@ func TestAcc_DeploymentWithEnvironment(t *testing.T) {
 			{
 				Config: testAccDeploymentConfig(projectSuffix, teamIDConfig(), `environment = {
                     FOO = "baz",
-                    BAR = "qux"
+                    BAR = "qux",
+                    BAZ = null
```

Comment:
Do we need to check that `BAZ` is not present on the output? I suppose this test would fail otherwise.

Reply:
The test failed before the changes b/c of the deserialization issue. It's a bit rubbish but the test helpers don't give a nice way to check for a `null` value on the output. I think just checking that it doesn't cause the provider to panic is as good as we can do (which is the purpose of this).